### ISSUE TITLE
Question about ISSAC seq computational pipeline

### ISSUE THREAD

**Issue by dKlee99** [Teichlab/scg_lib_structs#21]
Dear Dr. Chen,

I want to express my gratitude for developing the remarkable GitHub repository. I have a question regarding the ISSAC seq RNA pipeline: why wasn't the genefull option included, especially considering it is nucleus seq? Additionally, I noticed that the base mismatch option (--soloCBmatchWLtype 1MM) and --soloFeatures Gene GeneFull are not allowed. Could you provide insights into the reasoning behind these decisions?

Thank you for your time and consideration.

**Comment by dKlee99** [comment]
Also, was there any reasons why you  --clip3pNbases 116 option and make reads compared that 10x r1 uses around 90 bp ?

**Comment by dbrg77** [comment]
Hi @dKlee99 

Thanks for your question. Back in the time, we were not very familiar with all the STARsolo options, so we kept to the default most of the time. There was not any particular reasons. We probably overlooked that option.

Indeed, `GeneFull` is the preferred option considering the method is mainly profiling the poly-A RNA from the nucleus. We are now using `GeneFull` as well.

In terms of `--clip3pNbases 116`, this is due to the nature of the SHERRY library, which is generally short. We are doing 151bp PE sequencing. The majority of the bases at the 3' end are just adapters, so we only used the first 36 bp for the mapping. It seems the first 50 bp also work well.

I hope this helps.

Xi
